micmac_li3ds/import_orimatis.py

```diff
@@ -68,29 +74,35 @@ def get_parser(self, prog_name):
             '--indent', type=int,
             help='number of spaces for pretty print indenting')
         parser.add_argument(
-            'filename',
+            'filename', nargs='+',
             help='the orimatis file')
```

Comment:
You could mention that multiple orimatis files may be specificed on the command line.